packages/pigment-css-react/tests/theme/fixtures/themed-component.output.css

```diff
@@ -0,0 +1,28 @@
+.si9gu6v {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding: 0.75rem 1rem;
+  background-color: #ebf5ff;
+  border-radius: 0.25rem;
+  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.05);
+  letter-spacing: -0.025em;
+  font-weight: 600;
+}
+
+.si9gu6v-2 {
+  border: 2px solid #e9e9e9;
+}
+.si9gu6v-3 {
+  border-color: #ebf5ff;
+}
+.sbfbm5t {
+  font-size: 2rem;
+}
+.sbfbm5t-1 {
+  line-height: 1.7;
+}
+.s1xscf0o {
+  font-size: 1rem;
+  color: #6f7f95;
+}
```

Comment:
Theming works as expected. We can improve the readability of the generated classname later.

Eg. use `theme-` for styles that come from a theme.